src/driver/communication_interface.h

```diff
@@ -77,6 +78,9 @@ namespace franka_driver {
 struct RobotData {
   std::atomic<bool> has_robot_data;
   franka::RobotState robot_state;
+  int64_t current_plan_utime {};
+  int64_t plan_start_utime {};
+  double plan_completion_frac {-1.0};
```

Comment:
can be zero initialised since a meaningful value is always `0 < value <~ 1`

Reply:
Yes, but 0 can mean that the plan is starting to execute. I want to differentiate between the two

Reply:
you'd never get exactly 0 when the plan is being executed right?

Reply:
Fair. defaults to zero by default now